### ISSUE TITLE
Improve Widget hover border contrast

### ISSUE THREAD

**Issue by mlewand** [ckeditor/ckeditor-dev#1535]
## Are you reporting a feature request or a bug?

Bug

## Provide detailed reproduction steps (if any)

1. Open CKEditor instance with any widget, e.g. [Captioned Image SDK sample](https://sdk.ckeditor.com/samples/captionedimage.html).
2. Move mouse pointer over the widget, so that it's hovered.

### Expected result

The widget border has a nice contrast. Currently it's a bright yellow, which does not contrast that well with white background (which is the default content background we aim for).

Ofc the change should not be limited to widget itself, it should also include nested editables (like caption in case of Enhanced Image plugin).

### Actual result

Contrast should have improved contrast. I think CKEditor 5 team used a very nice color here, so I think we could use exact same color here:

See [CKE5 demo](https://ckeditor5.github.io/docs/nightly/ckeditor5/latest/examples/builds/classic-editor.html).

![CKEditor 5 hovered image shows a nice contrasting border](https://i.imgur.com/5LEaTEZ.jpg)
